### PR TITLE
Add publication_state predicate

### PR DESCRIPTION
**Pull request by jcoyne**
This would be the model for projecthydra/sufia#2452

### CODE REVIEW

File: spec/models/generic_work_spec.rb [hunk 1]
```diff
@@ -14,6 +14,16 @@
     it { is_expected.to eq 'curation_concerns_generic_work' }
   end
 
+  describe '#object_state' do
+    let(:work) { described_class.new }
+    let(:inactive) { ::RDF::URI('http://fedora.info/definitions/1/0/access/ObjState#inactive') }
```
Comment (cbeer) [inline spec/models/generic_work_spec.rb]: Not a blocker, but should we get these URIs pushed into `rdf-vocab` or similar?

Comment (jcoyne) [inline spec/models/generic_work_spec.rb]: I doubt that this is going to get published in `rdf-vocab` as it's not a widely used vocabulary.

Comment (mjgiarlo) [inline spec/models/generic_work_spec.rb]: Not *necessarily* true. We've already got https://github.com/ruby-rdf/rdf-vocab/blob/develop/lib/rdf/vocab/fcrepo4.rb in there, and there are a handful of Fedora/Hydra folks shepherding that gem along.

File: app/models/vocab/fedora_state.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+require 'rdf'
+module Vocab
+  class FedoraState < RDF::StrictVocabulary('http://fedora.info/definitions/1/0/access/')
```
Comment (mjgiarlo) [inline app/models/vocab/fedora_state.rb]: Minor points, but:

* Should we align the name of this class with the name of the ontology? It's the Fedora Resource Status ontology.
* I believe the URI here may be wrong. you're missing `ObjState#` on the end.

Comment (jcoyne) [inline app/models/vocab/fedora_state.rb]: I copied this class in from https://github.com/pulibrary/plum/blob/master/app/models/vocab/f3_access.rb#L2, so @escowles be aware that you may have the same issue.

**Pull request merged**